src/aspire/volume/volume.py

```diff
@@ -176,6 +179,44 @@ def _set_symmetry_group(self, value):
             )
         self._symmetry_group = value
 
+    def _symmetry_group_warning(self):
+        """
+        Warn when an operation has the potential to change the symmetry
+        of a volume or the alignment of symmetry axes.
+        """
+        msg = (
+            f"`symmetry_group` attribute is being set to `C1`. This operation may"
+            f" effect the symmetry (or symmetric alignment) of the volume. To reset the"
+            f" symmetry group run `self._set_symmetry_group('{self.symmetry_group}')`."
+        )
+        warnings.warn(msg, UserWarning, stacklevel=4)
+
+    def _result_symmetry(self, other=None):
+        """
+        Check if `other` will alter symmetry of `self` and return resulting symmetry_group.
+
+        :return: SymmetryGroup instance.
+        """
+        # No need to check for C1.
+        if isinstance(self.symmetry_group, IdentitySymmetryGroup):
+            return self.symmetry_group
+
+        # Warn and set to IdentitySymmetryGroup if `other` is symmetrically incompatible.
+        result_symmetry = self.symmetry_group
+
+        # Conditions of incompatibility.
+        self_transformation = other is None
+        incompat_syms = (
+            isinstance(other, Volume) and self.symmetry_group != other.symmetry_group
+        )
+        arbitrary_array = not isinstance(other, Volume) and hasattr(other, "__len__")
```

Comment:
What do you think about `np.array(other).size > 1`?

Reply:
Python has a way to get attr if it exists with a default when it doesn't.  Probably can just use that and get the situations that matter here.

https://docs.python.org/3/library/functions.html#getattr

Reply:
It appears using `getattr(np.array([123]), '__len__')` returns a method wrapper object instead of the actual length. I could follow it with a call, `getattr(np.array([123]), '__len__')()`, to get the length, but that wouldn't work for actual scalars.

Reply:
I'll try to come up with something else.

Reply:
you dont need to use __len__

Reply:
Ah yeah, ok. Using `getattr(other, size, 1) > 1` does the trick. Thanks.